plugin/builtin/sortordertransformer/go.mod

```diff
@@ -1,10 +1,13 @@
-module sigs.k8s.io/kustomize/plugin/builtin/legacyordertransformer
+module sigs.k8s.io/kustomize/plugin/builtin/sortordertransformer
 
-go 1.19
+go 1.18
 
 require (
 	github.com/pkg/errors v0.9.1
-	sigs.k8s.io/kustomize/api v0.11.5
+	github.com/stretchr/testify v1.8.0
+	sigs.k8s.io/kustomize/api v0.8.9
```

Comment:
This should use the latest version, i.e. [0.12.1](https://github.com/kubernetes-sigs/kustomize/releases/tag/api%2Fv0.12.1)

(kyaml is ok at 0.13.9 since we haven't updated anything to use 0.13.10 yet)